projects/plugins/jetpack/modules/search/instant-search/components/search-results.jsx

```diff
@@ -73,6 +74,18 @@ class SearchResults extends Component {
 				num,
 				corrected_query
 			);
+		} else if ( this.props.staticFilters && this.props.staticFilters.group_id !== '__NO_GROUP__' ) {
+			const group = getAvailableStaticFilters().filter( item => item.filter_id === 'group_id' );
+			const allP2 =
+				group.length === 1 && group[ 0 ].values
+					? group[ 0 ].values.filter( item => item.value !== '__NO_GROUP__' )
+					: {};
+			const p2Name = allP2[ 0 ]?.name ? allP2[ 0 ].name : __( 'All P2', 'jetpack' );
+			return sprintf(
+				_n( 'Found %s result in %s', 'Found %s results in %s', total, 'jetpack' ),
```

Comment:
Could you add a translator comment here? something like (I also updated the placeholders to be numbered and consider the fact that the number of results is numeric):

```suggestion
				/* translators: %1$s: number of results. - %2$s: site name. */
				_n( 'Found %1$d result in %2$s', 'Found %1$d results in %2$s', total, 'jetpack' ),
```

The same could be done just below.

Reply:
Sure thing, thanks for taking the time to suggest the change.